graphrag_sdk/document_loaders/pdf.py

```
@@ -15,11 +16,11 @@ def __init__(self, path: str) -> None:
         """
 
         try:
-            import pypdf
-        except ImportError:
-            raise ImportError(
+            __import__("pypdf")
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError(
                 "pypdf package not found, please install it with " "`pip install pypdf`"
-        )
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)